vignettes/Status.Rmd

```diff
@@ -21,17 +21,17 @@ This page will be updated frequently with information about the status of dataRe
 
 ### NWIS qw services
 
-Discrete water samples data are undergoing modernization, and NWIS services will no longer be updated with the latest data starting late February 2024, with a full decommission expected 6 months later. Learn more about the upcoming change and where to find the new samples data in our [blog](https://waterdata.usgs.gov/blog/changes-to-sample-data/).
```

Comment:
just to be super clear, can you update the text to read, 'and the NWIS services pulling 'qw' data will no longer...'